docs/src/tutorials/parameter_estimation.md

```diff
@@ -0,0 +1,102 @@
+# Parameter Estimation
+The parameters of a model, generated by Catalyst, can be fitted using various packages available in the Julia ecosystem. Refer [here](https://diffeq.sciml.ai/stable/analysis/parameter_estimation/) for a more detailed description. Below follows a quick tutorial of how a parameter set for a model can be fitted to data.
```

Comment:
Mention this tutorial uses [DiffEqFlux] point to docs.

Reply:
Along those lines perhaps also say that the `sciml_train` is using the `ADAM` gradient descent method?